spec/jobs/reports/monthly_key_metrics_report_spec.rb

```diff
@@ -14,15 +14,19 @@
   let(:document_upload_proofing_s3_path) { "#{report_folder}/document_upload_proofing.csv" }
   let(:account_deletion_rate_s3_path) { "#{report_folder}/account_deletion_rate.csv" }
   let(:total_user_count_s3_path) { "#{report_folder}/total_user_count.csv" }
+  let(:monthly_active_users_count_s3_path) { "#{report_folder}/monthly_active_users_count.csv" }
```

Comment:
Likewise, I think I may have accidentally zombied this back into existence in a merge conflict. 🧟 🎃